### PR TITLE
Documentation Updates

### PR DESCRIPTION
**Pull request by orifox**
Added some tables to both landing pages (GitHub and Readthedocs).

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -36,12 +36,147 @@ produced by JWST MIRI), along with 1D spectra extracted from the cube.
...
 
 Installing
 ----------
 For details on installing and using JDAViz, see the
 `Jdaviz documentation <https://jdaviz.readthedocs.io/en/latest/>`_.
 
+Help
+----------
+
+If you uncover any issues or bugs, you can open a GitHub ticket.  For faster responses, however, we encourage you to submit a JWST Help Desk Ticket: jwsthelp.stsci.edu
```
Comment (pllim) [inline README.rst:179]:
```suggestion
If you uncover any issues or bugs, you can `open a GitHub issue <https://github.com/spacetelescope/jdaviz/issues/new/choose>`_ if they are not already reported.  For faster responses, however, we encourage you to submit a `JWST Help Desk Ticket <https://jwsthelp.stsci.edu>`_.
```

**Review by pllim — COMMENTED** [review]
Nice table! However, I am not sure about repeating the same info at two different places; they are going to get out of sync.

Can we link README to the table section in `docs/index.rst` instead?

**Comment by orifox** [comment]
Ok, I think that looked good.  I committed that change.  Let me know if you approve.

**Comment by pllim** [comment]
Thanks! Do you plan to respond to my comment about the duplicated info in readme and index.rst ?

**Review by pllim — APPROVED** [review]
Table renders fine in https://jdaviz--698.org.readthedocs.build/en/698/ and the URL to it in README is expected to be broken until _after_ merge; nothing we can do about that.

I did add f2f2154cbaf5ba5a01447ad37da1f8becfb6a0c6 to this PR, to add back an accepted change that got reverted in the rebase process (forgot to `get fetch origin master` + `git rebase origin/master` first).

File: README.rst [hunk 2]
```diff
@@ -36,12 +36,22 @@ produced by JWST MIRI), along with 1D spectra extracted from the cube.
 **Imviz** provides visualization and quick-look analysis for 2D astronomical
 images.
 
+This tool is designed with instrument modes from the James Webb Space Telescope (JWST) in mind, but 
+the tool should be flexible enough to read in data from many astronomical telescopes.  The documentation provides a complete `table of all supported modes <https://jdaviz.readthedocs.io/en/latest/#jdaviz_instrument_table>`_.
```
Comment (pllim) [inline README.rst:40]: Actually, I think direct URL ref is different from Sphinx ref. See https://jdaviz--698.org.readthedocs.build/en/698/#jwst-instrument-modes-in-jdaviz

```suggestion
the tool should be flexible enough to read in data from many astronomical telescopes.  The documentation provides a complete `table of all supported modes <https://jdaviz.readthedocs.io/en/latest/#jwst-instrument-modes-in-jdaviz>`_.
```

**Review by rosteen — APPROVED** [review]
Looks good to me!

**Pull request merged**